64_FFT/app_resources/shader.comp.hlsl

```diff
@@ -0,0 +1,62 @@
+#include "common.hlsl"
+#include "nbl/builtin/hlsl/workgroup/fft.hlsl"
+#include "nbl/builtin/hlsl/bda/legacy_bda_accessor.hlsl"
+
+[[vk::push_constant]] PushConstantData pushConstants;
+
+// careful: change size according to Scalar type
+groupshared uint32_t sharedmem[4 * WorkgroupSize];
```

Comment:
make a thing like the workgroup scans have for working out the smem size with `nbl::hlsl::mpl`